packages/itwinui-css/src/transfer-list/transfer-list.scss

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@use '../input-container/mixins';
+@use './mixins' as *;
+
+.iui-transfer-list-wrapper {
+  display: grid;
+  grid-template-columns: 1fr auto 1fr;
```

Comment:
CSS suggestion: Maybe redo grid to have more than one row? Right now it messes up vertical centering when TransferList does not have message undeneath it.

Reply:
Genius. This has been added and the toolbar has been centered properly as a result

Reply:
Is it valid case for users to add message under the second column? Is it possible to do that easily with current API?
![image](https://github.com/iTwin/iTwinUI/assets/83585998/f929df50-4ad6-4f3c-9cbb-e9606b48421e)

Reply:
This is easily possible to do with our API as long as there is an empty `div` element to take the place of the middle column

![image](https://github.com/iTwin/iTwinUI/assets/78564221/c834fbd0-f85c-485b-9dd1-7db94c147a9e)
![image](https://github.com/iTwin/iTwinUI/assets/78564221/671f88bc-7d56-45d1-8d1a-de2c4085474a)